homeassistant/components/flume/sensor.py

```diff
@@ -40,25 +47,59 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     flume_token_file = hass.config.path("FLUME_TOKEN_FILE")
     time_zone = str(hass.config.time_zone)
     name = config[CONF_NAME]
+    include_devices = [
```

Comment:
We shouldn't add this. You're already adding a unique ID which allows users to disable the entity in the entity registry.

Reply:
Wasn’t aware of that functionality.  That’s much better.  Will remove

Reply:
Wasn’t aware of that functionality.  That’s much better.  Will remove

Reply:
Removed